subsys/bluetooth/audio/mcc_internal.h

```diff
@@ -84,21 +107,37 @@ struct mcs_instance_t {
 
 	struct bt_gatt_subscribe_params player_name_sub_params;
 	struct bt_gatt_subscribe_params track_changed_sub_params;
+#if defined(CONFIG_BT_MCC_READ_TRACK_TITLE)
 	struct bt_gatt_subscribe_params track_title_sub_params;
+#endif /* defined(CONFIG_BT_MCC_READ_TRACK_TITLE) */
```

Comment:
Even if you support reading the track title, it is still optional for the server to support notifications, and optional for the client to subscribe. 

> The Media Control Client may configure notifications of the Track Title characteristic (e.g., via the Client
Characteristic Configuration descriptor) so that the client can be alerted if the current track title changes.

We could / should consider adding support to enable/disable the subscription parts in separate (but related) Kconfig values